src/patterns/components/links/react/code/icon-text.hbs

```diff
@@ -1,4 +1,4 @@
-<SprkLink variant="has-icon" idString="link-4" href="#nogo">
+<SprkLink variant="has-icon" idString="link-4" href="#">
   {/* TODO: SVG Component*/}
```

Comment:
Do we still have a // TODO left in this file? We might need an issue created for this.

Reply:
You're right. Looks like the intent is to build a component that renders a svg. I'll let Stephanie know.